app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -698,6 +698,20 @@ class HomeFragment : Fragment() {
     }
 
     private fun navigateToSearch() {
+        // Dismisses the search dialog when the home content is scrolled
+        val recyclerView = sessionControlView!!.view
```

Comment:
Is there another way to access the `sessionControlView` without `!!`?

Reply:
We bang bang everywhere we reference this on home. Not great but 🤷